calculator.rb

```diff
@@ -0,0 +1,38 @@
+### Challenge 1 - Calculator
```

Comment:
Let's put this code inside of a class ```Calculator``` so that we can apply it to all instances of ```Calculator```.
```
class Calculator
	def initialize
	end
        # your methods here
end
```